Code/Tools/AssetProcessor/native/utilities/BatchApplicationManager.cpp

```diff
@@ -123,6 +123,9 @@ void BatchApplicationManager::MakeActivationConnections()
         {
             m_processedAssetCount++;
 
+            // If a file fails and later succeeds, don't count it as a failure
```

Comment:
Maybe expand your comment to explicitly note that this is to avoid failing jenkins jobs when intermediate assets cause transient failures.